src/api/services/enrichment/directPathMultipleValues.txt

```diff
@@ -5,7 +5,7 @@ plugin = analytics
 
 [assign]
 path = value
-value = get("value.[[SOURCE COLUMN]]").castArray().map(JSON.parse).reduce((acc, item) => item)
+value = get("value").update("[[SOURCE COLUMN]]", (item) => { try { return JSON.parse(item)} catch {return item } }).get("[[SOURCE COLUMN]]")
```

Comment:
How did you find this solution ? 😄

Reply:
Next dev on the project will be like.... 
![image](https://user-images.githubusercontent.com/2562270/148410426-1c722124-6c50-49bf-a3dc-0c1bd0e76c6a.png)

Can we leave an explanation somewhere?